src/docs/slot_specifications.md

```diff
@@ -0,0 +1,299 @@
+
+### 1.2 Language
+
+All MIxS terms attributes MUST be written in English.
+
+<!-- JFY comment: may be being a bit strict here, I guess you could have 'translated name' column or something like that, should rephrase to allow those exceptions -->
+
+## 2. Term structured naming
+
+### 2.1 Term name format
+
+The term (slot) name MUST be in [snake_case](https://en.wikipedia.org/wiki/Snake_case).
+
+All words must be lower case and underscores (`_`) MUST be used to separate words in the slot name.
+
+### 2.1 Term name length
+
+The term (slot) name must be a maximum of 20 characters in length as per INSDC guidelines ([https://www.insdc.org/submitting-standards/feature-table/#3.1](https://www.insdc.org/submitting-standards/feature-table/#3.1)).
+
+### 2.4 Term name uniqueness
+
+The term (slot) name MUST be unique within the MIxS LinkML model.
+
+### 2.5 Term name descriptiveness
+
+The term (slot) name MUST be descriptive of the data it is intended to hold.
+
+### 2.6 Term name abbreviation
+
+The term (slot) name SHOULD be a abbreviated form of the item (title) attribute.
+
+Examples:
+
+| Term Item / `title`                             | Structured comment name / `name` |
+| ----------------------------------------------- | -------------------------------- |
+| geographic location (country and/or sea,region) | `geo_loc_name`                   |
+| isolation and growth condition                  | `isol_growth_condt`              |
+| pcr conditions                                  | `pcr_cond`                       |
+| sample volume or weight for DNA extraction      | `samp_vol_we_dna_ext`            |
+| collection site geographic feature              | `coll_site_geo_feat`             |
+
+### 2.7 Term name common prefix
+
+When related to existing terms, the term (slot) name SHOULD use a common prefix that allow grouping of related terms.
+
+Examples:
+
+- Terms related to `sample` should use the prefix `samp_`.
+
+  | Term Item / `title`              | Structured comment name / `name` |
+  | -------------------------------- | -------------------------------- |
+  | sample storage temperature       | `samp_store_temp`                |
+  | sample storage duration          | `samp_store_dur`                 |
+  | sample volume for DNA extraction | `samp_vol_we_dna_ext`            |
+
+- Terms related to assembly metadata term (slots) should use the prefix `assembly_`.
+
+  | Term Item / `title`          | Structured comment name / `name` |
+  | ---------------------------- | -------------------------------- |
+  | name and version of assembly | `assembly_name`                  |
+  | assembly software            | `assembly_software`              |
+  | assembly quality             | `assembly_qual`                  |
+
+## 4. Term data types
+
+### 4.1 Term data types must be valid LinkML types
```

Comment:
I guess there is no direct MIxS equivelent to this? But I think it should be included in the table in preamble terminology table in the 'range' row. It just sort of jumped out of the blue here as a new concept. Maybe we want to change the header 4.1 to "The data types of a range must be valid LinkML types"

Reply:
I'm not sure now - your comment above seemed to imply that `expected value ` is the same as `range`, just `range` is much more restricted in the different types? Or is it not eqivalent then?